src/Templates/src/templates/maui-mobile/App.xaml

```diff
@@ -4,32 +4,6 @@
              xmlns:local="clr-namespace:MauiApp._1"
              x:Class="MauiApp._1.App">
     <Application.Resources>
```

Comment:
Should we be updating the `maui-blazor` template with similar changes, too?

Reply:
@davidortinau told me blazor already has this type of theming via bootstrap styles